routes/group_route.go

```diff
@@ -10,5 +11,5 @@ var (
 )
 
 func groupRoute(g *gin.RouterGroup) {
-	g.POST("/group", groupController.Create)
+	g.POST("/group", middleware.VerifyAuthenticated(), groupController.Create)
```

Comment:
Maybe one check we could add in the groupController.Create is that the bearer user is also in the usernames (right? it makes sense that you can't create a group but exclude yourself?).

Reply:
Yup makes sense, should probably be a separate ticket though right

Reply:
Sure, make an issue

Reply:
Want to just lump this into #30

Reply:
Sounds good.